### DISCUSSION TITLE
Let private window access cookies

### DISCUSSION THREAD

**Discussion by emanrdesu** [qutebrowser/qutebrowser#5888] [Q&A] [answered]
Is there a way to do this?
I can of course visit the site I want and then clear history.
But I dont want to clear history because there may be important history I need.
Maybe there should be a setting in qute:settings to allow private window to use cookies

**Comment by The-Compiler** [comment, marked as answer]
Can you elaborate on what your goal is? You might want to set `completion.web_history.exclude` perhaps?

**Reply by emanrdesu** [reply]
So, if I have cookies on a site, and I open a private window, the cookies will not be used. I want to be able to use cookies on private windows. Or have the option to at least.

**Reply by The-Compiler** [reply]
I understand, but why? What's the point of using private mode but with cookies? Like I said, if it's just about not adding things to history completion, you might want `completion.web_history.exclude` instead.
